test/fail_compilation/fail19890b.d

```diff
@@ -0,0 +1,8 @@
+// PERMUTE_ARGS:
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail19890b.d(8): Error: `void[$n$]` size 1 * $n$ exceeds 0x7fffffff size limit for static array
```

Comment:
I don't think your `$n$` PR went through.

Reply:
I missed an LU. My pr was for handling hex values, which means that the test comparison will still fail after I add it.

I think that it may be better to handle pcre regex instead though.